src/term.rs

```diff
@@ -108,6 +108,17 @@ impl<V: Version> Term<V> {
     }
 }
 
+// Other useful functions.
+impl<V: Version> Term<V> {
+    /// Check if a term allows a given version.
+    pub fn contains(&self, version: &V) -> bool {
```

Comment:
The `term` module was not initially thought of as being part of the public API but making the `Term` type public was necessary for error reporting (no solution) so we made everything except the `Term` enum `pub(crate)`.

Reply:
PPS: this already exists as `accept_version`, which is obviously bad naming since you rewrote it. I like `contains` better I think

Reply:
Good catch. Fixed.